README.md

```diff
@@ -101,3 +97,24 @@ Then run the application (here on Android):
 ```sh
 npm run cordova-android
 ```
+
+## How to build the Chrome extension ``hoverpad.crx`` file?
+
+Assuming the ``chromium-browser`` command is available in your ``PATH``:
```

Comment:
I don't have the command. Can we link `chromium-browser` to npm, or docs, or the project homepage for Chromium-browser?